### PR TITLE
2 more CLB cards

### LINKED ISSUE #846: Earth Tremor

### LINKED ISSUE #921: Patron of the Arts

### PR DESCRIPTION
**Pull request by JohnWilliams77**
- closes #921 
- closes #846

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/earth_tremor.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Earth Tremor
+ManaCost:3 R
+Types:Instant
+A:SP$ DealDamage | Cost$ 3 R | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | ReplaceDyingDefined$ Targeted | SpellDescription$ CARDNAME deals damage to target creature or planeswalker equal to the number of lands you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/earth_tremor.txt:4]: Remove ReplaceDyingDefined$

File: forge-gui/res/cardsfolder/upcoming/patron_of_the_arts.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Patron of the Arts
+ManaCost:2 R
+Types:Creature Dragon Noble
+PT:3/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield or dies, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | Secondary$ True | TriggerDescription$ When CARDNAME enters the battlefield or dies, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ TriggeredCardController
+DeckHas:Ability$Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/patron_of_the_arts.txt:8]: DeckHas:Ability$Sacrifice|Token & Type$Treasure|Artifact

File: forge-gui/res/cardsfolder/upcoming/earth_tremor.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Name:Earth Tremor
+ManaCost:3 R
+Types:Instant
+A:SP$ DealDamage | Cost$ 3 R | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SpellDescription$ CARDNAME deals damage to target creature or planeswalker equal to the number of lands you control.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/earth_tremor.txt:4]: please remove Cost from this line

**Review by Northmoc — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**